chaoskube/chaoskube.go

```diff
@@ -197,7 +200,8 @@ func (c *Chaoskube) DeletePod(victim v1.Pod) error {
 		return nil
 	}
 
-	err := c.Client.CoreV1().Pods(victim.Namespace).Delete(victim.Name, nil)
+	deleteOptions := metav1.DeleteOptions{GracePeriodSeconds: &c.GracePeriod}
+	err := c.Client.CoreV1().Pods(victim.Namespace).Delete(victim.Name, &deleteOptions)
```

Comment:
This is the main change, where I've added a `DeleteOptions` containing the selected grace period to the call to `pod.Delete()`